### PR TITLE
Added docmentation for changing the default translation namespace alias

### PR DESCRIPTION
**Pull request by dantleech**
See https://github.com/doctrine/DoctrinePHPCRBundle/pull/185

### CODE REVIEW

File: bundles/phpcr_odm/configuration.rst [hunk 1]
```diff
@@ -451,6 +451,7 @@ not configure anything here, the ODM services will not be loaded.
                 auto_generate_proxy_classes: "%kernel.debug%"
                 proxy_dir: "%kernel.cache_dir%/doctrine/PHPCRProxies"
                 proxy_namespace: PHPCRProxies
+                translation_namespace: i18n
```
Comment (dbu) [inline bundles/phpcr_odm/configuration.rst]: the default namespace is `phpcr_locale`, or am i wrong?

Comment (dantleech) [inline bundles/phpcr_odm/configuration.rst]: Your right.

**Comment by dantleech** [comment]
Updated.

File: bundles/phpcr_odm/configuration.rst [hunk 2]
```diff
@@ -572,6 +584,15 @@ Change folder where proxy classes are generated.
...
 
+``namespaces``
+""""""""""""""
+
+This configuration section is intended to allow you to customize the
+PHPCR namespaces used by PHPCR-ODM. Currently it is only possible to
+set the alias used by the translation strategy.
+
+
+
```
Comment (dbu) [inline bundles/phpcr_odm/configuration.rst]: too many blank lines - did you mean to add more here? seems good enough to me but then lets remove the lines.

File: bundles/phpcr_odm/configuration.rst [hunk 3]
```diff
@@ -451,6 +451,9 @@ not configure anything here, the ODM services will not be loaded.
                 auto_generate_proxy_classes: "%kernel.debug%"
                 proxy_dir: "%kernel.cache_dir%/doctrine/PHPCRProxies"
                 proxy_namespace: PHPCRProxies
+                namespaces:
+                    translation:
+                        alias:                phpcr_locale
```
Comment (dbu) [inline bundles/phpcr_odm/configuration.rst]: can you remove the whitespace here? this is more indented than it should be

Comment (dbu) [inline bundles/phpcr_odm/configuration.rst]: the whitespace between "alias:" and "phpcr_locale" i mean.

**Comment by dbu** [comment]
otherwise this looks fine. feel free to merge away when the whitespace is removed.

File: bundles/phpcr_odm/configuration.rst (hunk 3 above)
Comment (dantleech) [inline bundles/phpcr_odm/configuration.rst]: well this is copy-paste from the `config:dump-reference` command, see below (`type`) for another example of more whitespace than needed

Comment (dbu) [inline bundles/phpcr_odm/configuration.rst]: but its inconsistent with the lines above where we do not align the values.

**Comment by dantleech** [comment]
Whitespace fixed.

**Pull request merged**